src/components/Datatable/Datatable.tsx

```diff
@@ -55,6 +56,7 @@ const defaultControlsConfig: ControlsConfig<Record<string, unknown>> = {
     placeholder: 'Search',
     onSearch: noop,
   },
+  onSuggestionsFetch: noop,
```

Comment:
Why not in `searchConfig` it is only useful for the search feature not for the whole controls